docs/using-connector.md

```diff
@@ -486,17 +487,50 @@ df = sqlContext.read \
       .format("org.apache.spark.sql.riak") \
       .schema(schema) \
       .load(ts_table_name) \
-      .filter(s"time >= CAST(111111 AS TIMESTAMP) AND time <= CAST(555555 AS TIMESTAMP) AND col1 = 'val1'")
+      .filter(s"time >= CAST(10000 AS TIMESTAMP) AND time < CAST(210000 AS TIMESTAMP) AND col1 = 'val1'")
 
 ```
 
 The initial range query will be split into 5 subqueries (one per each partition) as follows:
-* ```time >= CAST(111111 AS TIMESTAMP) AND time < CAST(222222 AS TIMESTAMP) AND col1 = 'val1'```
-* ```time >= CAST(222222 AS TIMESTAMP) AND time < CAST(333333 AS TIMESTAMP) AND col1 = 'val1'```
-* ```time >= CAST(333333 AS TIMESTAMP) AND time < CAST(444444 AS TIMESTAMP) AND col1 = 'val1'```
-* ```time >= CAST(444444 AS TIMESTAMP) AND time < CAST(555555 AS TIMESTAMP) AND col1 = 'val1'```
-* ```time >= CAST(555555 AS TIMESTAMP) AND time < CAST(555556 AS TIMESTAMP) AND col1 = 'val1'```
+* ```time >= CAST(10000 AS TIMESTAMP) AND time < CAST(50000 AS TIMESTAMP) AND col1 = 'val1'```
+* ```time >= CAST(50000 AS TIMESTAMP) AND time < CAST(90000 AS TIMESTAMP) AND col1 = 'val1'```
+* ```time >= CAST(90000 AS TIMESTAMP) AND time < CAST(130000 AS TIMESTAMP) AND col1 = 'val1'```
+* ```time >= CAST(130000 AS TIMESTAMP) AND time < CAST(170000 AS TIMESTAMP) AND col1 = 'val1'```
+* ```time >= CAST(170000 AS TIMESTAMP) AND time < CAST(210000 AS TIMESTAMP) AND col1 = 'val1'```
 
+Additional option **spark.riak.partitioning.ts-quantum** can be passed to notify spark riak connector of quantum size. Then if automalically created subranges break 5 quanta limitation, initial range will be split into ~4*quantum subranges and resulting subranges will then be grouped to form required number of partitions.
```

Comment:
An additional option **spark.riak.partitioning.ts-quantum** can be passed to notify the Spark-Riak Connector of the quantum size. If the automatically created subranges break the 5 quanta limitation, the initial range will be split into ~4 quantum subranges and the resulting subranges will then be grouped to form the required number of partitions.